### ISSUE TITLE
Header position

### ISSUE THREAD

**Issue by nicopasso** [diogobernardino/WilliamChart#30]
Hi all,
I've a little problem with the headers of the StackBarChart. the position of the headers is wrong and it's not centered horizontally in the bar. I can't find a way to move them.
The picture below shows my situation:
![header_wrong_position](https://cloud.githubusercontent.com/assets/1663205/5743855/4184244a-9c1c-11e4-9bdb-b3b8ec88edd6.png)

How can fix this? 

Thanks
Nico

**Comment by diogobernardino** [comment]
How are you getting that coordinate (top-left of the bar), can you show that part of your code?

**Comment by nicopasso** [comment]
I get the coordinates with these two lines of code:
```java
mStackBarChart.setBarSpacing(Tools.fromDpToPx(95));
mStackBarChart.setBorderSpacing(Tools.fromDpToPx(65));
```
and then I set the header:
```java
mStackBarChart.setHeader(0, number + " €");
```

**Comment by diogobernardino** [comment]
Not sure if understood. Those first lines are both sets not gets and they are not related with bar positioning. And the method ```setHeader``` you mention is not even implemented.

**Comment by nicopasso** [comment]
uhm...
I've inherited the project by other developers...I don't really now where they found your chart library but in the manifest the package name is 'com.db.williamchart' and I googled it and found this repo.
Am I wrong? Is this the wrong repo?

Anyway, about the code:
with setBarSpacing and setBorderSpacing I was able to place the bars in the graph correctly. 
And the headers don't need a positioning method. 
With "setHeader", the library do the work by itself.
